packages/node_modules/@ciscospark/react-redux-spark/src/component.js

```diff
@@ -34,40 +34,45 @@ class SparkComponent extends Component {
       props.updateSparkStatus({authenticating: spark.isAuthenticating});
     });
 
-    if (spark.mercury) {
+    if (spark.mercury && !spark.mercury.connected) {
```

Comment:
This section looks right, but take maybe take a look at both the example-phone app and the web client's mercury handling just in case.